File: src/dotnet/Models/EndpointInfo.cs
namespace Opsbridge.R18A6.Models
{
    public class EndpointInfo
    {
        public string Controller { get; set; }
        public string Path { get; set; }
        public string HttpMethod { get; set; }

        public EndpointInfo() { }
        public EndpointInfo(string controller, string path, string method)
        {
            Controller = controller;
            Path = path;
            HttpMethod = method;
        }
    }
}

File: src/main/java/com/opsbridge/r18a6/model/EndpointInfo.java
package com.opsbridge.r18a6.model;

public class EndpointInfo {

    private String controller;
    private String path;
    private String httpMethod;

    public EndpointInfo() {}

    public EndpointInfo(String controller, String path, String httpMethod) {
        this.controller = controller;
        this.path = path;
        this.httpMethod = httpMethod;
    }

    public String getController() { return controller; }
    public String getPath() { return path; }
    public String getHttpMethod() { return httpMethod; }

    @Override
    public String toString() {
        return controller + " " + httpMethod + " " + path;
    }

}
